tableaudocumentapi/connection.py

```diff
@@ -4,6 +4,61 @@
 #
 ###############################################################################
 
+KNOWN_DB_CLASSES = ('msaccess',
```

Comment:
Hahahaha. But getting them all lined up took the most time for this PR! (And writing the regex to extract them)

constants.py? Or something more specific?